### PR TITLE
[iOS/tvOS] unify setting bundle identifier

### PR DESCRIPTION
**Pull request by kambala-decapitator**

### CODE REVIEW

**Comment by sy6sy2** [comment]
Thank you! I will apply it on the corresponding commit ;-)

**Comment by sy6sy2** [comment]
Changes done on SylvainCecchetto:ATV-cmake-generation

File: cmake/scripts/ios/ArchSetup.cmake [hunk 1]
```diff
@@ -3,6 +3,7 @@ if(NOT CMAKE_TOOLCHAIN_FILE)
 endif()
 
 
+set(PLATFORM_BUNDLE_IDENTIFIER "${APP_PACKAGE}-${CORE_PLATFORM_NAME_LC}")
```
Comment (sy6sy2) [inline cmake/scripts/ios/ArchSetup.cmake:6]: After testing it seems that `APP_PACKAGE` is not set here because I obtain `-tvos` for the bundle identifier.

Comment (kambala-decapitator) [reply cmake/scripts/ios/ArchSetup.cmake:6]: oh, too bad, didn't expect this... Then don't include it now, I'll find a better place to set.

**Pull request closed without merging**